src/victory-primitives/clip-path.js

```diff
@@ -28,28 +29,25 @@ export default class ClipPath extends React.Component {
     translateY: PropTypes.number
   };
 
+  static defaultProps = {
+    translateX: 0,
```

Comment:
Most of the props for clipPath end up being controlled by a parent component like VictoryZoomContainer or VictoryTransition so that the behaviors are correct. However, we want things like clipPadding etc to be overridable by a user setting props directly on the clipPath component, so the parent components will set props like `defaults({}, clipPathComponent.props, {parent controlled props})`.